SUMMARY.md

```diff
@@ -16,6 +16,7 @@
 
     - [Swap - Buy agEUR](/guides/app-guides/getageur.md)
     - [Borrow / Leverage](/guides/app-guides/borrow-ageur.md)
+    - [UniV3 incentives in Angle](/guides/app-guides/univ3-lp.md)
```

Comment:
Let's put this guide as a subpage somewhere of the governance guides here: https://docs.angle.money/overview/guides/veangle-guides/incentives

Like it's good if we can stick to one page one guide with potentially subguides, and we link this guide to the upper incentives page